pkg/services/store/object/utils.go

```diff
@@ -0,0 +1,14 @@
+package object
+
+func SameObject(a *GRN, b *GRN) bool {
```

Comment:
since the protobuf struct has an embedded mutex, we always need to pass this by reference

This means we can't use the simple equals 😦